File: MusicPlayer.v2/app/src/main/java/com/example/musicplayerv2/Themes.java
package com.example.musicplayerv2;

import android.content.Context;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;

/**
 * Created by ${Aina} on 11.11.2018.
 */
public class Themes {

    public static int getTheme(Context context) {
        SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(context);
        String thisTheme = sp.getString("this", "current");
        int themeId = R.style.AppTheme;
        switch (thisTheme) {
            case "First":
                themeId = R.style.first;
                break;
            case "Second":
                themeId = R.style.second;
                break;
            case "Third":
                themeId = R.style.third;
                break;
        }
        return themeId;
    }
}

File: MusicPlayer.v2Kotlin/app/src/main/java/com/example/musicplayerv2/Themes.kt
package com.example.musicplayerv2

import android.content.Context
import android.content.SharedPreferences
import android.preference.PreferenceManager

/**
 * Created by ${Aina} on 11.11.2018.
 */
object Themes {

    fun getTheme(context: Context): Int {
        val sp = PreferenceManager.getDefaultSharedPreferences(context)
        val thisTheme = sp.getString("this", "current")
        var themeId = R.style.AppTheme
        when (thisTheme) {
            "First" -> themeId = R.style.first
            "Second" -> themeId = R.style.second
            "Third" -> themeId = R.style.third
        }
        return themeId
    }
}
